src/sql/workbench/api/node/extHostConnectionManagement.ts

```diff
@@ -61,7 +61,7 @@ export class ExtHostConnectionManagement extends ExtHostConnectionManagementShap
 		return this._proxy.$getUriForConnection(connectionId);
 	}
 
-	public $connect(connectionProfile: azdata.IConnectionProfile, saveConnection: boolean = false, showDashboard: boolean = false): Thenable<azdata.ConnectionResult> {
+	public $connect(connectionProfile: azdata.IConnectionProfile, saveConnection: boolean = true, showDashboard: boolean = true): Thenable<azdata.ConnectionResult> {
```

Comment:
We probably don't want `showDashboard` defaulting to true. One case for example would be connections for CMS.

Reply:
It was true before your CMS changes - and you specifically pass in false when doing connect for CMS 

https://github.com/Microsoft/azuredatastudio/blob/fix/oeTestBreak/extensions/cms/src/apiWrapper.ts#L213

```
await azdata.connection.connect(Utils.toConnectionProfile(connection), false, false).then(async (result) 
```

So I don't see any issue here - although I'm happy to test specific scenarios if you have any you think would be broken/changed by this.